fineract-provider/src/main/java/org/apache/fineract/infrastructure/configuration/domain/ConfigurationDomainServiceJpa.java

```diff
@@ -608,4 +608,11 @@ public Integer retriveIvaConfiguration() {
         return value;
     }
 
+    @Override
+    public LocalDate retrievePenaltyStartDate() {
+        final String propertyName = "penalty-start-date";
+        final GlobalConfigurationPropertyData property = getGlobalConfigurationPropertyData(propertyName);
```

Comment:
check if config is enabled @hagafrank

Reply:
if config is disabled  , we can return a very early date

Reply:
Thanks @oluexpert99, I have added code to handle case where the config is disabled